app/views/admin/access_controls/_table.haml

```diff
@@ -1,29 +1,32 @@
-= render 'common/pagination_top', item_name: 'access control'
-.table-responsive
-  %table.table.table-striped
-    %thead
-      %tr
-        %th User Group
-        %th Role
-        %th Collection
-        %th
-    %tbody
-      - @access_controls.each do |acl|
+= render 'common/pagination_top', item_name: 'access control', filter_partial: 'filter'
+- if @pagy.count.positive?
```

Comment:
This render can probably go inside the `if` block, it is redundant:
<img width="834" alt="Screenshot 2024-03-19 at 11 26 31 AM" src="https://github.com/greenriver/hmis-warehouse/assets/5333982/1eb53ee4-9603-4d76-803b-42c98d0da7c3">